generator/generator_config.textproto

```diff
@@ -3683,6 +3683,7 @@ service {
     "WriteObject"
   ]
   omit_repo_metadata: true
+  omitted_rpcs: ["GetServiceAccount", "CreateHmacKey", "DeleteHmacKey", "GetHmacKey", "ListHmacKeys", "UpdateHmacKey", "DeleteNotificationConfig", "GetNotificationConfig", "CreateNotificationConfig", "ListNotificationConfigs"]
```

Comment:
nit: this might be a bit easier to read broken onto multiple lines

Reply:
Done.